pkgs/games/cataclysm-dda/default.nix

```diff
@@ -0,0 +1,51 @@
+{ fetchFromGitHub, stdenv, makeWrapper, pkgconfig, ncurses, lua, SDL2, SDL2_image, SDL2_ttf,
+SDL2_mixer, freetype, gettext }:
+
+let architecture =
+      if stdenv.system == "i686-linux" then
+        "linux32"
+      else if stdenv.system == "x86_64-linux" then
+        "linux64"
+      else
+        abort "currently only linux 32-bit and 64-bit are supported";
+
+in stdenv.mkDerivation rec {
+  version = "0.C";
+  name = "cataclysm-dda-${version}";
+
+  src = fetchFromGitHub {
+    owner = "CleverRaven";
+    repo = "Cataclysm-DDA";
+    rev = "${version}";
+    sha256 = "03sdzsk4qdq99qckq0axbsvg1apn6xizscd8pwp5w6kq2fyj5xkv";
+  };
+
+  buildInputs = [ makeWrapper pkgconfig ncurses lua SDL2 SDL2_image SDL2_ttf SDL2_mixer freetype gettext ];
+
+  patchPhase = ''
+    patchShebangs .
+    substituteAllInPlace lang/compile_mo.sh \
+      --replace msgfmt ${gettext}/msgfmt
+    sed -i -e 's|DATA_PREFIX=$(PREFIX)/share/cataclysm-dda/|DATA_PREFIX=$(PREFIX)/share/|g' Makefile
+  '';
+
+  buildPhase = ''
+    make PREFIX=$out LUA=1 TILES=1 SOUND=1 RELEASE=1 USE_HOME_DIR=1 NATIVE=${architecture} -j$NIX_BUILD_CORES
+  '';
+
+  installPhase = ''
+    make PREFIX=$out LUA=1 TILES=1 SOUND=1 RELEASE=1 USE_HOME_DIR=1 install
```

Comment:
Couldn't you pass those to `installFlags` and move the call to `wrapProgram` to `postInstall`? That way we can use the default `installPhase`.